components/vtpm2-server/startup_tpm.sh

```diff
@@ -7,6 +7,11 @@ export TPM_INTERFACE_TYPE=${TPM_INTERFACE_TYPE:-socsim}
 export TPM_COMMAND_PORT=${SWTPM_SERVER_PORT}
 export TPM_DEVICE=${TPM_DEVICE:-/dev/tpm0}
 export TPM_PERSISTENT_KEY_INDEX=${TPM_PERSISTENT_KEY_INDEX:-81230001}
+export TPM_OWNER_PASSWORD_FORMAT=${TPM_OWNER_PASSWORD_FORMAT:-hex}
```

Comment:
Maybe set the default not to `hex` but to empty string and require the hex to be passed in from the outside? Otherwise this may easily mess up a password like `password`.